### DISCUSSION TITLE
KTX 2.0

### DISCUSSION THREAD

**Discussion by unicomp21** [google/filament#4134] [Ideas]
Does filament support this?
https://cgsociety.org/news/article/5106/khronos-group-announces-new-3d-texture-format

**Comment by unicomp21** [comment]
If not, is there an existing 3D texture format I can use?  for creating 3D tiled textures?  any examples i can look at?

**Reply by romainguy** [reply]
We don't provide a library to use KTX 2.0 yet but Filament offers all the APIs you need to support KTX 2.0 yourself (you can use Khronos' reference library too).

**Reply by unicomp21** [reply]
should I put something on the backlog?
